Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -73,6 +73,18 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath)
 %% 1.   Load PWI
 fprintf('%s\n','Loading PWI & M0 images');
 
+if isfield(x,'UseBasilQuantification') & x.UseBasilQuantification
+    % For BASIL we want the 4D PWI even though we will end up with
+    % 3D output - so identify which one we need and substitute it
+    if strcmp(PWI_Path, x.P.Path_PWI)
```

Comment:
I would replace this PWI path by the 4D one, in the parent ASL wrapper `xASL_module_ASL`, rather than here. Similar to how this is done in line 422 in `xASL_module_ASL` for the `x.Q.SaveCBF4D` option (but then using `x.UseBasilQuantification`)

Reply:
I can do this - but since there is already conditional code for `UseBasilQuantification` in `xASL_wrp_Quantify` I thought it better to keep it all there. I guess by moving it you would avoid having to compare paths directly, was this the main idea or were there other reasons for putting this into the parent ASL module?

Reply:
That makes sense indeed, sorry I probably have mistaken the level. I changed the codebase now that there should always be PWI4D, so it is safe to just use `x.P.Path_PWI4D` as input for BASIL, without checking it's existence or definition right?